src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/processors/OffenderCreatedEventProcessor.kt

```diff
@@ -35,10 +35,6 @@ class OffenderCreatedEventProcessor(
       },
       onFailure = {
         log.error("Error retrieving new offender detail: ${it.message}")
-        telemetryService.trackEvent(
-          TelemetryEventType.DELIUS_OFFENDER_READ_FAILURE,
-          mapOf("CRN" to crnIdentifier?.value),
-        )
```

Comment:
Again, don't see the need for an AppInsights event here - this is a problem with an internal API call